ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -430,4 +430,27 @@ def get_deviceset_count():
     sc = get_storage_cluster()
     return int(sc.get().get('items')[0].get('spec').get(
         'storageDeviceSets')[0].get('count')
+               )
+
+
+def get_all_storageclass(all_ns=False):
+    """
+    Function for getting all storageclass
+
```

Comment:
add the parameter in the description

Reply:
parameter was removed duo to Pratik's comment about storage classes don't have a namespace